logistic_requisition/model/logistic_requisition.py

```diff
@@ -340,7 +340,7 @@ def button_view_source_lines(self):
     @api.multi
     def _get_relevant_purchases(self):
         sourced_lines = self.line_ids.filtered(
-            lambda rec: rec.state == 'Sourced'
+            lambda rec: rec.state in ('Sourced', 'Quoted')
```

Comment:
I don't get why we use display value instead of database value does it really work ? If yes, what about translations ?